docs/source/cli/graph-construction/distributed/gsprocessing/input-configuration.rst

```diff
@@ -104,6 +104,10 @@ objects:
          -  e.g. ``"files": ['path/to/edge/type/']``
          -  This option allows for concise listing of entire types and
             would be preferred. All the files under the path will be loaded.
+         -  You can also provide a wildcard pattern, e.g.
+            ``"files": ['path/to/edge-type*.parquet']`` would match
```

Comment:
I would suggest for the example here we use `type` instead of `edge-type`, otherwise there may be users asking why only supporting edges not nodes.

Reply:
This doc is in the edges documentation though? For the nodes docs we say the same applies as for edges.